tests/combobox-select-only/data/tests.csv

```diff
@@ -1,33 +1,33 @@
-14,Open a collapsed select-only combobox to the first option,voiceover_macos,interaction,open collapsed select-only combobox to first option,moveFocusToCombobox,sets focus on the combobox,aria-expanded listbox aria-labelledby option aria-selected,"With focus on the combobox, open the combobox and set focus to the first option.",State of the combobox (expanded) is conveyed,Role of the popup ('listbox') is conveyed,Name of the popup ('Favorite Fruit') is conveyed,Text of the focused option in the popup ('Choose a Fruit') is conveyed,2:State of the focused option in the popup (selected) is conveyed,Position of the focused option in the popup (1) is conveyed,Number of options in the popup (13) is conveyed
+1,Navigate forwards to a collapsed select-only combobox in reading mode,"JAWS,NVDA",reading,navigate forwards to collapsed select-only combobox,moveFocusBeforeCombobox,sets focus on a link before the Combobox,combobox aria-labelledby aria-expanded,With the reading cursor on the 'Navigate forwards from here' link, navigate to the 'Favorite Fruit' Combobox.,Role 'combobox' is conveyed,Name 'Favorite Fruit' is conveyed,Text of the selected option ('Choose a Fruit') is conveyed,State of the combobox (collapsed) is conveyed,,,
+2,Navigate backwards to a collapsed select-only combobox in reading mode,"JAWS,NVDA",reading,navigate backwards to collapsed select-only combobox,moveFocusAfterCombobox,sets focus on a link after the Combobox,combobox aria-labelledby aria-expanded,With the reading cursor on the 'Navigate backwards from here' link, navigate to the 'Favorite Fruit' Combobox.,Role 'combobox' is conveyed,Name 'Favorite Fruit' is conveyed,Text of the selected option ('Choose a Fruit') is conveyed,State of the combobox (collapsed) is conveyed,,,
+3,Navigate forwards to a collapsed select-only combobox in interaction mode,"JAWS,NVDA",interaction,navigate forwards to collapsed select-only combobox,moveFocusBeforeCombobox,sets focus on a link before the Combobox,combobox aria-labelledby aria-expanded,With focus on the 'Navigate forwards from here' link, navigate to the 'Favorite Fruit' Combobox.,Role 'combobox' is conveyed,Name 'Favorite Fruit' is conveyed,Text of the selected option ('Choose a Fruit') is conveyed,State of the combobox (collapsed) is conveyed,,,
+4,Navigate backwards to a collapsed select-only combobox in interaction mode,"JAWS,NVDA",interaction,navigate backwards to collapsed select-only combobox,moveFocusAfterCombobox,sets focus on a link after the Combobox,combobox aria-labelledby aria-expanded,With focus on the 'Navigate backwards from here' link, navigate to the 'Favorite Fruit' Combobox.,Role 'combobox' is conveyed,Name 'Favorite Fruit' is conveyed,Text of the selected option ('Choose a Fruit') is conveyed,State of the combobox (collapsed) is conveyed,,,
```

Comment:
`With focus on the 'Navigate backwards from here' link, navigate to the 'Favorite Fruit' Combobox.` should be wrapped in double quotes due to mid sentence comma.